.ci/ci.yml

```diff
@@ -57,6 +57,18 @@ resources:
           uri: git@github.com:((github-account.username))/puppet-google-sql.git
           private_key: ((repo-key.private_key))
 
+    - name: chef-compute-intermediate
```

Comment:
It would be nice after this CL is in you create 1 PR that adds a single product. That would become the canonical PR do add new products going forward. Right now this PR does much more than that.